src/main/scala/org/scalacheck/Prop.scala

```diff
@@ -806,6 +806,7 @@ object Prop {
 
   /** Universal quantifier for an explicit generator. Shrinks failed arguments
    *  with the default shrink function for the type */
+  @deprecated("Use 'forAllNoShrink', shrinking will be disabled for Gen by default in the next release", "1.14.1")
```

Comment:
I suck with useful error messages. Suggestions welcome for what might be more appropriate/useful/readable.

Reply:
As I mentioned in #440, it would be better to provide users with `forAllShrink`, since `forAllNoShrink` does not provide shrinking.  

This requires adding all the same arities of `forAllShrink(Gen)` as exist for `forAll(Gen)`.  This would probably end up needing to be released as 1.15.0.

The wording of the deprecation should probably be something like, "Shrink will be disabled when given explicit Gen, use forAllShrink instead", giving:

```
[warn] method forAll in object Prop is deprecated (since 1.15.0): 
[warn] Shrink will be disabled when given explicit Gen, use forAllShrink instead
```